doc/sources/_README.md

```diff
@@ -0,0 +1,269 @@
+[![LedgerSMB](https://circleci.com/gh/ledgersmb/LedgerSMB/tree/master.svg?style=svg)](https://circleci.com/gh/ledgersmb/LedgerSMB/tree/master)
+[![Lgtm total alerts](https://img.shields.io/lgtm/alerts/g/ledgersmb/LedgerSMB.svg?logo=lgtm&logoWidth=18)](https://lgtm.com/projects/g/ledgersmb/LedgerSMB/alerts/)
+[![GPLv2 Licence](https://badges.frapsoft.com/os/gpl/gpl.png?v=103)](https://opensource.org/licenses/GPL-2.0/)
+[![Coverage Status](https://coveralls.io/repos/github/ledgersmb/LedgerSMB/badge.svg?branch=master)](https://coveralls.io/github/ledgersmb/LedgerSMB?branch=master)
+[![Docker](https://img.shields.io/docker/pulls/ledgersmb/ledgersmb.svg)](https://hub.docker.com/r/ledgersmb/ledgersmb/)
+[![Language grade: JavaScript](https://img.shields.io/lgtm/grade/javascript/g/ledgersmb/LedgerSMB.svg?logo=lgtm&logoWidth=18)](https://lgtm.com/projects/g/ledgersmb/LedgerSMB/context:javascript)
+[![Mentioned in Awesome <awesome-selfhosted>](https://awesome.re/mentioned-badge.svg)](https://github.com/Kickball/awesome-selfhosted#enterprise-resource-planning)
+
+
+
+# SYNOPSIS
+
+LedgerSMB is a free integrated web application accounting system, featuring
+double entry accounting, budgeting, invoicing, quotations, projects, timecards,
+inventory management, shipping and more ...
+
+The UI allows world-wide accessibility; with its data stored in the
+enterprise-strength `PostgreSQL` open source database system, the system is known
+to operate smoothly for businesses with thousands of transactions per week.
+Screens and customer visible output are defined in templates, allowing easy and
+fast customization. Supported output formats are PDF, CSV, HTML, ODF and more.
+
+Directly send orders and invoices from the built-in e-mail function to your
+customers or RFQs (request for quotation) to your vendors with PDF attachments.
+
+
+# System requirements
+
+Note that these are the system requirements for LedgerSMB <<LSMB_VERSION>>, the current
+development version. Please check the system requirements for [the 1.7 old stable
+version](https://github.com/ledgersmb/LedgerSMB/tree/1.7#system-requirements)
+and [the 1.8 version](https://github.com/ledgersmb/LedgerSMB/tree/1.8#system-requirements).
+
+## Server
+
+* `Perl 5.24+`
+* `PostgreSQL 9.6+`
+* Web server (e.g. `nginx`, `Apache`, `lighttpd`, `Varnish`)
+
+The web external server is only required for production installs;
+for evaluation purposes a simpler setup can be used, as detailed
+below.
+
+## Client
+
+A `Dojo` compatible web browser is all that's required on the client;
+it includes the recent versions of `Chrome`, `FireFox`, `Opera`,
+`MS Internet Explorer` and a wide range of mobile browsers.
+Please note that LedgerSMB
+explicitly doesn't make any attempts to support Internet Explorer.
+
+The tables below show the browsers currently supported. their earliest date
```

Comment:
small typo: `show` -> `shows`

Reply:
oh, and I think you'll want to change the dot to a comma.

Reply:
Period changed. No typo, tables are plural.

Reply:
duh! sorry about that.